app/src/main/java/org/openobservatory/ooniprobe/model/database/Result.java

```diff
@@ -94,9 +94,21 @@ public long countOkMeasurements() {
 	public long countAnomalousMeasurements() {
 		return SQLite.selectCountOf().from(Measurement.class).where(Measurement_Table.result_id.eq(id), Measurement_Table.is_rerun.eq(false), Measurement_Table.is_done.eq(true), Measurement_Table.is_failed.eq(false), Measurement_Table.is_anomaly.eq(true)).count();
 	}
+	
+	private Measurement getFirstMeasurement() {
+		return SQLite.select().from(Measurement.class).where(Measurement_Table.result_id.eq(id), Measurement_Table.is_rerun.eq(false)).orderBy(Measurement_Table.start_time, true).querySingle();
+	}
+
+	private Measurement getLastMeasurement() {
+		return SQLite.select().from(Measurement.class).where(Measurement_Table.result_id.eq(id), Measurement_Table.is_rerun.eq(false)).orderBy(Measurement_Table.start_time, false).querySingle();
```

Comment:
Can you wrap these lines to a maximum column width to make it more readable?

Reply:
I also verified that `querySingle` does the right thing of applying LIMIT 1, and it does according to: https://dbflow.gitbook.io/dbflow/v/develop/usage2/usage/retrieval#synchronous-retrieval